docbook/gnc-locale-C.dtd

```diff
@@ -39,171 +53,555 @@
 
 <!-- guimenu/guisubmenu/guimenuitem to be used as names in text -->
 <!-- <!ENTITY % gmi.xxx "<guimenuitem><accel></accel></guimenuitem>"> -->
-<!-- <!ENTITY gmi. "%gmi.;"> -->
+<!-- <!ENTITY gmi.xxx.yy-zz "%gmi.xxx.yy-zz;"> -->
+<!-- Description: 
+     gm:  Entry in the menubar
+     gmi: Every menu entry, also in submenus
```

Comment:
I wonder why menu entry get's abbreviated by gmi? Its derived from `GuiMenuItem?
Perhaps we should use such terms then here too.